Core.DomainModel/User.cs

```diff
@@ -125,38 +125,57 @@ public override string ToString()
 
         public bool IsGlobalAdmin { get; set; }
 
-        public bool IsReadOnly {
-            get
-            {
-                return OrganizationRights.Any(
-                    right => (right.Role == OrganizationRole.ReadOnly) &&
-                             (right.OrganizationId == DefaultOrganizationId.GetValueOrDefault()));
-            }
+        public bool IsReadOnly => IsReadOnlyInOrg(DefaultOrganizationId.GetValueOrDefault());
+
+        public bool IsReadOnlyInOrg(int organizationId)
+        {
+            return OrganizationRights.Any(
+                right => (right.Role == OrganizationRole.ReadOnly) &&
+                         (right.OrganizationId == organizationId));
         }
 
         public override bool HasUserWriteAccess(User user)
         {
-            if (IsReadOnly) {
-                return (Id == user.Id) || base.HasUserWriteAccess(user);
-            }else
+            if (IsReadOnly)
             {
-                return IsReadOnly;
+                return (Id == user.Id) || base.HasUserWriteAccess(user);
             }
+
+            return IsReadOnly;
         }
 
         public bool IsInContext(int organizationId)
         {
             return DefaultOrganizationId == organizationId;
         }
 
-        public bool IsLocalAdmin
+        public bool IsLocalAdmin => IsLocalAdminInOrg(DefaultOrganizationId.GetValueOrDefault());
```

Comment:
Jeg synes det virker forkert at et bruger objekt selv skal kunne svare på spørgsmål omkring localadmin og organistoriske tilhørsforhold .. bør det som minimum ikke være placeret i en helper?

Reply:
Jo.. men jeg vil ikke ændre alle de steder det bliver brugt i dag (mange). Det er lidt ude for scope. Min ændring i denne fil bestod af at gøre det muligt at spørge på en konkret org i stedet for den hard codede afhængighed til DefaultOrg

Reply:
Ok - måske kan man udflytte metoderne til en extension klasse (this User user). På den måde bevares syntaxen, men intentionen viser vejen til et senere udflyt?

Reply:
Jeg forstår godt din kommentar, men hele problematikken omkring domænemodellen eller manglen på samme er dækket af en anden story. Jeg vil ikke rykke for meget rundt på den eksisterende kode i konteksten af dette PR. Jeg synes ikke det løser noget at det kommer i en extension. Der er et utal af stedet hvor man spørger på alt mulig mærkeligt ligesom IContextAware, som er et problem, der heller ikke skal løses i dette PR :-)

Reply:
Desuden eksponeres IsLocalAdmin hele vejen til frontend via OData så man kan ikke bare flytte den ud i en extension uden at skulle i front-end og lave ballade. Derfor bør denne opgave løses som en del af remodellerings-opgaven :-)